modules/apps/portal/portal-context/src/main/java/com/liferay/portal/context/internal/CompanyInvocationContextProviderImpl.java

```diff
@@ -54,6 +55,17 @@ public boolean isPresent() {
 		return true;
 	}
 
+	@Override
+	public SafeCloseable setCurrent(Company company) {
```

Comment:
Why do you think it is necessary to have a 'setter'? 
Couldn't the provider just be a way to retrieve the context object without supporting setting it?
Who are you thinking will be invoking this method?